README.md

```diff
@@ -28,6 +28,19 @@ template_foo: "oof"
 template_bar: "baz"
 ```
 
+## Return Values
```

Comment:
Not sure if this is the best name, IMHO `Variables the Template Role Defines` matches better since there is no real concept of return values in Ansible for roles, is there?

Reply:
there isn't indeed, it is just in analogy with modules where it exists.

Reply:
see https://github.com/linux-system-roles/template/pull/33/commits/9f6eb0a1fee95458d1e238c3277d6b1a550f98eb